ErrorBlocks.md

```diff
@@ -0,0 +1,156 @@
+
+<details><p><summary>Notes</summary>
+- These volumes could also be created in another CAD program and then assigned a material in the exact same way, but this is an example of how to create these volumes in Trelis and Cubit.
+<br/></br>
+- Most likely the volumes will be slightly off centered in the x direction, but the larger bricks will be centered with one another. This may vary based on the initial CAD program.
+<br/></br>
+- To see the blocks better you can select the wireframe option in the top toolbar. It is directly to the right of the undo arrow on the toolbar. Also you can click on the axis on the bottom left of the volume viewing area to get a straight on (x,y) view.
+</p></details>
+
+#### Imprinting and Merging Volumes in Cubit/Trelis
+_Imprinting and Merging surfaces accelerates the DAGMC process by combining surfaces._
+
+  1. Imprint and Merge all shared surfaces in the model.
+     * In command panel select __Geometry -> Volume -> Imprint Merge__ and select __Imprint and Merge__ from the dropdown menu. Then type "all" into the __Volume ID(s)__ box and click __Apply__.
+     * In command line type `imprint volume all` and then enter. Then type `merge volume all`.
+     * _Now all surfaces that previously overlapped have been merged into one surface shared by both volumes._
+
+<details><p><summary>Notes</summary>
+- The default tolerated distance for imprinting/merging surfaces is 5.0e-4 units.
+<br/>
+- Combining surfaces cuts two surfaces and two sets of tetrahedron mesh areas down to one, decreasing computational power required as well as eliminating possible errors where particles exist simultaneously in two surfaces.
+</p></details>
+
+#### Assigning Materials in Cubit/Trelis
+_Most commands for this section are best implemented through the command line. More general instructions can
+be found_ [here](http://svalinn.github.io/DAGMC/usersguide/codes/dag-mcnp.html).
+
+  1. The general command for assigning Materials/Densities in Cubit/Trelis
+  `group "mat:[assigned material number]/rho:[assigned density]" add
+  vol [volume numbers]` where the density is in [atoms/barn-cm] for positive values and [g/cc] for negative values.
+  2. The blocks will be given two different materials and then the void and the graveyard volumes will be assigned. These material assignments will depend on the material library you create/use for the project.
+     * To assign one block to the first material use `group "mat:1/rho:0.0223" add vol 1`
+     * To assign the rest of the blocks to the second material use `group "mat:2/rho:0.03" add vol 2 to 4`
+     * To assign the void volume to vacuum use `group "mat:Vacuum" add vol 5`
+     * Finally to assign the graveyard volume use `group "mat:Graveyard" add vol 6`
+
+#### Exporting file to .h5m format
+_This plugin translates the CAD file to something that can be used by DAGMC. Instructions as to how to install this plugin can be found_ [here](https://svalinn.github.io/DAGMC/install/plugin.html).

```

Comment:
Note, plugin has to be installed prior to opening and doing all this stuff in cubit/trelis. If someone gets through all the imprint/merge steps and finds they need to install the plugin, they will have to exit and re-open. This section on installing the plugin may be better to have as a first step before they even open the program.

Reply:
(At least I think this is true, but maybe @gonuke can confirm/deny that behavior)